boston-housing/index.js

```diff
@@ -94,17 +109,32 @@ export const run = async (model) => {
   });
 
   await ui.updateStatus('Running on test data...');
-  const result =
-      model.evaluate(testData.data, testData.target, {batchSize: BATCH_SIZE});
+  const result = model.evaluate(
+      tensors.testFeatures, tensors.testTarget, {batchSize: BATCH_SIZE});
   const testLoss = result.get();
   await ui.updateStatus(
       `Final train-set loss: ${trainLoss.toFixed(4)}\n` +
-      `Final validation-set loss: ${valLoss.toFixed(4)}\n` +
       `Test-set loss: ${testLoss.toFixed(4)}`);
 };
 
+export const computeBaseline = () => {
+  const avgPrice = tf.mean(tensors.trainTarget);
+  console.log(`Average price: ${avgPrice.dataSync()}`);
+  const baseline = tf.mean(tf.pow(tf.sub(tensors.testTarget, avgPrice), 2));
+  console.log(`Baseline loss: ${baseline.dataSync()}`);
+  const baselineMsg = `Baseline loss (meanSquaredError) is ${
+      baseline.dataSync()[0].toFixed(2)}`;
+  ui.updateBaselineStatus(baselineMsg);
+};
+
 document.addEventListener('DOMContentLoaded', async () => {
-  await data.loadData();
-  await ui.updateStatus('Data loaded!');
+  await bostonData.loadData();
+  ui.updateStatus('Data loaded, converting to tensors');
+  arraysToTensors();
+  ui.updateStatus(
+      'Data is now available as tensors.\n' +
+      'Click a train button to begin.');
+  ui.updateBaselineStatus('Estimating baseline loss');
+  await computeBaseline();
```

Comment:
`computeBaseline` does not return a promise. We shouldn't `await` it

Reply:
Done.